<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DimmingTools</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Form{
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.482773, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(217, 237, 255, 255), stop:1 rgba(255, 255, 255, 255));

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#Form{
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.482773, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(217, 237, 255, 255), stop:1 rgba(255, 255, 255, 255));

}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Time/ms</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TimeData">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
border:1px solid rgb(213, 213, 213);
border-radius:5px;
padding:2px 4px;
}</string>
           </property>
           <property name="maxLength">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_1">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Light/nit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LightData">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
border:1px solid rgb(213, 213, 213);
border-radius:5px;
padding:2px 4px;
}</string>
           </property>
           <property name="maxLength">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="WritePoint">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
border:1px solid rgb(194, 194, 194);
border-radius:10px;
padding:2px 4px;
}
QPushButton:hover {
background-color: rgba(190, 227, 255,200);
}
QPushButton:pressed {
 background-color: rgba(85, 170, 255, 200);
}</string>
           </property>
           <property name="text">
            <string>Wrtie</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Revocation">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
border:1px solid rgb(194, 194, 194);
border-radius:10px;
padding:2px 4px;
}
QPushButton:hover {
background-color: rgba(190, 227, 255,200);
}
QPushButton:pressed {
 background-color: rgba(85, 170, 255, 200);
}</string>
           </property>
           <property name="text">
            <string>Revocation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Save">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
border:1px solid rgb(194, 194, 194);
border-radius:10px;
padding:2px 4px;
}
QPushButton:hover {
background-color: rgba(190, 227, 255,200);
}
QPushButton:pressed {
 background-color: rgba(85, 170, 255, 200);
}</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>NowPoint</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PointNumber">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
border:1px solid rgb(213, 213, 213);
border-radius:5px;
padding:2px 4px;
}</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCustomPlot" name="Waveform" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>450</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
